examples/long-running/long-running-script.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use DDTrace\SpanData;
+
+// This part is required for long running processes
+\DDTrace\trace_function('repetitive_function', function (SpanData $span, $args) {
+    $span->service = 'my_service';
```

Comment:
In the extension we should make this default to `DD_SERVICE` if it's not set, but until then it might be nice to use `DD_SERVICE` here somehow.

Reply:
I propose to add `getenv('DD_SERVICE')` to this example. Maybe we can update once we define the final api to access our config utility functions for manual tracing. What do you think?